CareKit/CareCard/OCKCareCardViewController.m

```diff
@@ -130,6 +130,10 @@ - (void)prepareView {
     if (!_headerView) {
         _headerView = [[OCKCareCardTableViewHeader alloc] initWithFrame:CGRectZero];
     }
+    if ([_headerTitle length] == 0) {
+        _headerTitle = OCKLocalizedString(@"CARE_CARD_HEADER_TITLE", nil);
+    }
+    _headerView.title = _headerTitle;
```

Comment:
Apparently it ignores the length check and the label disappears, if I change according to this. Any suggestion (I am new to Objective-C...)?